### PR TITLE
Suggestion to make this ext match the original request

### PR DESCRIPTION
**Pull request by FlyThePlanet**
Hi Kailey,

Here is my suggestion to make this extension match the original request...
https://www.phpbb.com/community/viewtopic.php?p=15497631#p15497631

I've added the 2nd field to enable/disable the functionality group by group and then allow using "0" for "unlimited".
I've also added the French translation.
And I've tested everything, it seems to work fine, icons & rights.

-=et=- (on phpBB)

### CODE REVIEW

File: language/en/common.php [hunk 1]
```diff
@@ -38,6 +38,8 @@
 //
 
 $lang = array_merge($lang, [
-	'GROUP_EDIT_TIME'			=> 'Group limit editing time',
-	'GROUP_EDIT_TIME_EXPLAIN'	=> 'Limits the group time available to edit a new post. Setting the value to 0 allows unlimited edit time. <b>Warning:</b> This extension overrides the board-wide setting!',
+	'GROUP_ENABLE_EDIT_TIME'			=> 'Enable specific time management for post editing',
+	'GROUP_ENABLE_EDIT_TIME_EXPLAIN'	=> 'Allows to manage a specific time for the group members. This setting overcomes the limit time to edit posts set for ordinary forum users.',
+	'GROUP_EDIT_TIME'					=> 'Maximum time to edit posts',
```
Comment (kinerity) [inline language/en/common.php:43]: Please change this and the line below it back to what I had. The only thing that should be removed is everything after (and including) the "Warning" text.

File: event/main_listener.php [hunk 2]
```diff
@@ -91,99 +91,127 @@ public static function getSubscribedEvents()
...
-
 	private function get_group_id_ary()
 	{
 		$sql = 'SELECT g.group_id, g.group_edit_time
 			FROM ' . $this->tables['groups'] . ' g
 			LEFT JOIN ' . $this->tables['user_group'] . ' ug
 				ON ug.group_id = g.group_id
-			WHERE ug.user_id = ' . (int) $this->user->data['user_id'];
+			WHERE g.group_enable_edit_time = 1 AND ug.user_id = ' . (int) $this->user->data['user_id'];
+
```
Comment (kinerity) [inline event/main_listener.php:213]: Please remove the blank lines.

Comment (FlyThePlanet) [reply event/main_listener.php]: Done.

File: language/en/common.php (hunk 1 above)
Comment (FlyThePlanet) [reply language/en/common.php:43]: Not sure what you ask me. Are you talking only about the lines 43/44 or all the 41/44?
I suggest you merge the PR if you want to do so and then adjust the wording to whatever you want.
My PR is just a suggestion but it's of course your extension.

**Pull request merged**